core/api/serializers.py

```
@@ -213,14 +204,16 @@ def update(self, instance, validated_data):
                     **video_data
                 )
 
-        # Update tags if provided
+    def _update_tags(self, instance, tags_data):
```

Ruff (0.8.2)

Missing return type annotation for private function `_update_tags`

Add return type annotation: `None`

(ANN202)